.golangci.yml

```diff
@@ -47,6 +47,9 @@ issues:
   exclude-use-default: false
   exclude:
     - "package-comments"
+  exclude-rules:
+    - text: 'shadow: declaration of "err" shadows declaration at'
+      linters: [ govet ]
```

Comment:
allows err shadowing, for example, the `err` in this statement:

`formStmt, err := tx.PrepareNamed`

is disallowed due to shadowing, leading to alternative naming, such as `formErr`. We should be able to use the same name for errors, especially in different scopes, but I have limited the linter exceptions to `err` _ONLY_ - the linter will still catch other instances of shadowing

```go
err = stmt.Get(&createdFeedback, feedback)

	if err != nil {
		appcontext.ZLogger(ctx).Error(
			fmt.Sprintf("Failed to create TRB feedback with error %s", err),
			zap.String("user", feedback.CreatedBy),
		)
		return nil, err
	}

	// If the feedback requests edits, update the form status to "in progress"
	if formToUpdate != nil {
		formStmt, err := tx.PrepareNamed(`
			UPDATE trb_request_forms
			SET
				status = :status,
				modified_by = :modified_by,
				modified_at = CURRENT_TIMESTAMP
			WHERE trb_request_id = :trb_request_id
			RETURNING *;
		`)

// ........
```

Additionally, this linter exception allows us to have

```go
data, err := something()
if err != nil {
    return err
}

// this if statement receives its own err variable in a new scope
if err := somethingElse(); err != nil {
    return err
}

```